core/src/test/java/io/grpc/DeadlineTest.java

```diff
@@ -69,15 +71,19 @@ public void shortDeadlineEventuallyExpires() throws Exception {
     assertTrue(deadline.timeRemaining(TimeUnit.NANOSECONDS) > 0);
     assertFalse(deadline.isExpired());
     Thread.sleep(101);
+
     assertTrue(deadline.isExpired());
-    assertEquals(0, deadline.timeRemaining(TimeUnit.NANOSECONDS), delta);
+    long actualDelta = deadline.timeRemaining(NANOSECONDS);
+    assertTrue(maxDelta + " < " + actualDelta, maxDelta >= actualDelta);
```

Comment:
Fair enough. Since the same pattern is repeated for quite a few times, wouldn't it be better to define a helper function for it? How about an `assertEquals(long, long, long)` in a utility class?